torchio/transforms/augmentation/spatial/random_affine.py

```diff
@@ -125,39 +115,86 @@ def get_params(
     @staticmethod
     def get_scaling_transform(
             scaling_params: List[float],
+            center_lps: Optional[TypeTripletFloat] = None,
             ) -> sitk.ScaleTransform:
         # scaling_params are inverted so that they are more intuitive
         # For example, 1.5 means the objects look 1.5 times larger
         transform = sitk.ScaleTransform(3)
         scaling_params = 1 / np.array(scaling_params)
         transform.SetScale(scaling_params)
+        if center_lps is not None:
+            transform.SetCenter(center_lps)
         return transform
 
     @staticmethod
     def get_rotation_transform(
             degrees: List[float],
+            center_lps: Optional[TypeTripletFloat] = None,
             ) -> sitk.Euler3DTransform:
         transform = sitk.Euler3DTransform()
         radians = np.radians(degrees)
         transform.SetRotation(*radians)
+        if center_lps is not None:
+            transform.SetCenter(center_lps)
         return transform
 
+    def apply_transform(self, sample: Subject) -> dict:
+        sample.check_consistent_shape()
+        scaling_params, rotation_params = self.get_params(
+            self.scales, self.degrees, self.isotropic)
+        for image in sample.get_images(intensity_only=False):
+            if image[TYPE] == LABEL:
+                interpolation = Interpolation.NEAREST
+            else:
+                interpolation = self.interpolation
+
+            if image.is_2d():
+                scaling_params[0] = 1
+                rotation_params[-2:] = 0
+
+            if self.use_image_center:
+                center = image.get_center(lps=True)
+            else:
+                center = None
```

Comment:
if I correctly understand 
center = [0, 0, 0] would led to the same result right ?

Reply:
Yes. Do you think I should make it more explicit?